Library/Formula/libksba.rb

```diff
@@ -1,8 +1,8 @@
 class Libksba < Formula
   desc "X.509 and CMS library"
   homepage "https://www.gnupg.org/related_software/libksba/index.en.html"
-  url "ftp://ftp.gnupg.org/gcrypt/libksba/libksba-1.3.3.tar.bz2"
```

Comment:
Did some `gnupg.org` (URLs) vs. `www.gnupg.org` (homepages) inconsistency fixup. They may be further synced by using the latter for URLs as well.